source/beamline/VESPERS/VESPERS13ElementGeDetector.h

```diff
@@ -0,0 +1,71 @@
+#ifndef VESPERS13ELEMENTGEDETECTOR_H
+#define VESPERS13ELEMENTGEDETECTOR_H
+
+#include "beamline/AMXRFDetector.h"
+
```

Comment:
It's actually the same detector, but it will probably use it's own EPICS PVs.  I think the main difference is that @epengr has a bit more stuff in the XRF detector view than VESPERS does, typically.